<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Sistema de Gestión de Restaurante&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>188</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Seleccione una opcion</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>188</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnVerEmpleados">
        <property name="text">
         <string>Ver lista de empleados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVerDisponibilidad">
        <property name="text">
         <string>Verificar disponibilidad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAsignarEmpleados">
        <property name="text">
         <string>Asignar empleados a días del mes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVerAsignaciones">
        <property name="text">
         <string>Ver asignaciones</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSalir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWidgetEmpleados">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>31</y>
        <width>431</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QComboBox" name="comboRol">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Pastelero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cocinero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manager</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnCheckDisponibilidad">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Revisar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboDia">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Lunes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Martes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Miercoles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jueves</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viernes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sabado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Domingo</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="textVerificarDisp">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextBrowser" name="textAsignacionesResult">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTextBrowser" name="listWidgetAsignaciones">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>451</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxDiaAsignacion">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>42</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConfirmarDiaAsignacion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dia</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
